src/config.c

```diff
@@ -97,7 +97,9 @@ enftun_config_init(struct enftun_config* config)
 
     config->trusted_ifaces = calloc(2, sizeof(char*));
 
-    config->ra_period = 10 * 60 * 1000; // milliseconds
+    config->ra_period         = 10 * 60 * 1000; // milliseconds
+    config->heartbeat_period  = 12 * 1000;
```

Comment:
Are these in milliseconds?

If so, let's make the `heartbeat_period` be `5 * 60 * 1000` (5 minutes) and the `heartbeat_timeout` be `15 * 1000` (15 seconds).